x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,307 @@
+you are able to batch.  The following plot compares the best-case
+situation from earlier in this chapter, which included a change output,
+to an otherwise-identical changeless transaction.
+
+![Plot of normal best case against a changeless best
+case](img/p2wpkh-batching-changeless.png)
+
+Although the benefit may be small in percentage terms, the cost of
+implementing changeless transactions may also be small.  Your goal is to
+find a single input you own whose value is equal to or slightly larger
+than the total value of the outputs you want to send.  A simple
+algorithm would be to wait until you have a minimum number of payments
+to batch and check to see if you have an input that is equal to or
+slightly larger than that.  If not, wait until you have another payment
+to send, combine that with the earlier payments, and try again.  Keep
+trying until you reach a limit (e.g. it's time to send the oldest
+payment) or the aggregate output value is larger than your largest
+input.
+
+For more information about changeless transactions, including exactly
+what "slightly larger than the output value" means, see the
+corresponding section in the chapter about [coin selection
+strategies][].
+
+### Replace-by-Fee (RBF)
+
+RBF is typically promoted as a strategy for increasing the fees paid in
+a transaction so that it confirms sooner, but it also allows for adding
+outputs to a transaction, making it suitable for adding even more
+payments to a previously-sent transaction.
+
+However, RBF transactions must pay a higher feerate than the original
+transaction.  This means replacing a large batch payment costs more than
+replacing a small independent payment.  Since fee batching benefits more
+from larger transactions, the cost increases of RBF act in direct
+opposition to the fee savings of payment batching.
+
+Looking at the savings rate charts above, we can consider some examples.
+If you sent a single payment and want to turn that into a batch
+containing two payments, you'll expect to save 40% from batching in the
+best case---so paying a fee increase of anything up to 40% can save you
+money overall.  Alternatively, if you already have a batch of ten
+payments and are considering adding an eleventh, the extra savings is
+only about 1%, so it might not be worth it if RBF requires you increase
+the overall feerate of the transaction by more than 1%.
+
+However, there is a time when it's guaranteed to be more efficient to
+add new payments to an existing transaction: when you plan to fee bump
+that existing transaction anyway.  If you sent a payment some time ago
+and are planning to use RBF to increase its fee rate, you should also
+add any additional queued payments to the replacement transaction so
+that they gain from batched efficiency.
+
+For more information about RBF, see the chapter about [fee bumping][].
+
+## Recommendations summary
+
+1. Try to create systems where your users and customers don't expect
```

Comment:
Should we qualify this only pertains to non high frequency spenders? And should it be the top recommendation if it only pertains to a subset of the audience?

Reply:
Is this related to your comment at https://github.com/bitcoinops/scaling-book/pull/18#discussion_r260560866 or is it separate?  I personally think everyone using onchain payments should try to get receivers to expect maximum delay, as (1) it's fundamental to waiting for confirmation, (2) it greatly helps with reducing tx priority (feerate), and (3) it makes batching practical even for lower-frequency spenders.

Reply:
It was, and I will retract my comment based on the responses.

Reply:
I agree that this should be number one recommendation for everyone, regardless of how frequently they spend